codebase/superdarn/src.lib/tk/elevation.1.0/include/elevation.h

```diff
@@ -28,8 +28,8 @@ along with this program. If not, see <https://www.gnu.org/licenses/>.
 #ifndef _ELEVATION_H
 #define _ELEVATION_H
 
-double elevation_v2(struct FitPrm *prm, double psi_obs);
-double elevation(struct FitPrm *prm, double phi0);
-double elev_goose(struct FitPrm *prm, double phi0);
+double elevation(struct FitPrm * prm, double phi0);
+double elevation_v2(struct FitPrm * prm, double psi_obs);
+double elev_goose(struct FitPrm * prm, double phi0);
```

Comment:
```suggestion
double elevation(struct FitPrm *prm, double phi0);
double elevation_v2(struct FitPrm *prm, double psi_obs);
double elev_goose(struct FitPrm *prm, double phi0);
```

This seems like an unnecessary stylistic change from the rest of the RST.